File: OldProject/SpaceFist/SpaceFist/SpawnZone.cs
using Microsoft.Xna.Framework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SpaceFist
{
    /// <summary>
    /// Represents a region of a level where enemies will may spawn.
    /// </summary>
    public class SpawnZone
    {
        private int   count;
        private int   left;
        private int   right;
        private int   top;
        private int   bottom;
        private Point center;

        public int   Count  { get { return count;  } }
        public int   Left   { get { return left;   } }
        public int   Right  { get { return right;  } }
        public int   Top    { get { return top;    } }
        public int   Bottom { get { return bottom; } }
        public Point Center { get { return center; } }

        public SpawnZone(int count, int left, int right, int top, int bottom, Point center)
        {
            this.count  = count;
            this.left   = left;
            this.right  = right;
            this.top    = top;
            this.bottom = bottom;
            this.center = center;
        }
    }
}

File: core/src/com/spacefist/SpawnZone.java
package com.spacefist;

import com.badlogic.gdx.math.Vector2;

/**
 * Represents a region of a level where enemies will may spawn.
 */
public class SpawnZone {
    private int count;
    private int left;
    private int right;
    private int top;
    private int bottom;
    private Vector2 center;

    public SpawnZone(int count, int left, int right, int top, int bottom, Vector2 center) {
        this.count  = count;
        this.left   = left;
        this.right  = right;
        this.top    = top;
        this.bottom = bottom;
        this.center = center;
    }

    public int getCount() {
        return count;
    }

    public int getLeft() {
        return left;
    }

    public int getRight() {
        return right;
    }

    public int getTop() {
        return top;
    }

    public int getBottom() {
        return bottom;
    }

    public Vector2 getCenter() {
        return center;
    }
}